app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < RedisStore
 
   # vaafi attributes
   attribute :edipi
-  attribute :issue_instant
+  attribute :login_time
```

Comment:
We can change this to `attribute :last_signed_in, Common::UTCTime` to enforce that the attribute is a UTC time.